typescript/test/test_application_js/integration-test.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const util = require('util');
+const child_process = require('child_process');
+const exec = util.promisify(child_process.exec);
+const fs = require('fs-extra');
+const path = require('path');
+const SHOWCASE_SERVER = path.join(__dirname, 'showcase-server');
+const serverProcess = require( './server');
+const GAPIC_SHOWCASE_VERSION = '0.5.0';
+const OS = 'uname';
```

Comment:
This won't work (it's just literally `"uname"`, not its output). In node, you can take a look at `process.platform` and `process.arch`.